docs/schema/concepts/by-theme/transportation/shape-connectivity.mdx

```diff
@@ -196,18 +189,7 @@ back toward the start of the segment.
 
 </div>
 
-### Directionality
```

Comment:
Is the directionality topic covered anywhere right now?

Reply:
I removed this because it was specifically talking about directionality in lanes and we removed lanes from the transportation schema in the last release. I'll add a new directionality section in the docs before the next release.